testsuite/config/__init__.py

```diff
@@ -39,9 +39,9 @@ def __init__(self, name, default, **kwargs) -> None:
             & Validator("service_protection.authorino.oidc_url", must_exist=True)
         ),
         DefaultValueValidator(
-            "tracing.collector_url", default=fetch_service("jaeger-collector", protocol="rpc", port=4317)
+            "tracing.collector_url", default=fetch_service("tempo-sample", protocol="rpc", port=4317)
         ),
-        DefaultValueValidator("tracing.query_url", default=fetch_route("jaeger-query", force_http=True)),
+        DefaultValueValidator("tracing.query_url", default=fetch_route("tempo", force_http=True)),
```

Comment:
I would like to see PR into tools which replaces jaeger with tempo before doing this

Reply:
That is the reason why is this PR marked as draft